docs/source/index.rst

```diff
@@ -20,7 +20,7 @@ Earth surface dynamics. It contains:
 In addition Landlab contains a set of Jupyter notebook tutorials providing
 an introduction to core concepts and examples of use.
 
-Landlab was desiged for disciplines that quantify Earth surface dynamics such
```

Comment:
if this typo is here it is probably also in the readme.rst.

right now I have some text in these two documents mirroring each other, might you make sure that (where appropriate the readme also changes).

Reply:
You are correct! Fixed it.